src/common/com/intellij/plugins/haxe/ide/generation/CreateGetterSetterFix.java

```diff
@@ -137,7 +137,7 @@ private static void buildSetter(StringBuilder result, String name, String typeTe
   }
 
   private static void build(StringBuilder result, String name, String typeText, boolean isGetter) {
-    result.append("public function ");
+    result.append("private function ");
```

Comment:
Seems like public is a better default, though.  The point of a getter or setter is to expose the variable through the class interface, isn't it?